plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -2551,6 +2552,16 @@ public OptionalLong executeDelete(ConnectorSession session, ConnectorTableHandle
         return OptionalLong.of(deletedRecords - removedPositionDeletes - removedEqualityDeletes);
     }
 
+    @Override
+    public void truncateTable(ConnectorSession session, ConnectorTableHandle tableHandle)
+    {
+        IcebergTableHandle table = checkValidTableHandle(tableHandle);
+        Table icebergTable = catalog.loadTable(session, table.getSchemaTableName());
+        DeleteFiles deleteFiles = icebergTable.newDelete()
+                .deleteFromRowFilter(alwaysTrue());
```

Comment:
Do we need another test besides `io.trino.testing.BaseConnectorTest#testTruncateTable` to check also against `$files` metadata table that it is empty?
Or is it rather an overkill...